scripts/loud.coffee

```diff
@@ -41,7 +44,12 @@ module.exports = (robot) ->
           res.send "Loud deleted."
         else
           res.send "Couldn't find that loud."
-    
+
+      when 'ban'
+        if data in robot.brain.get('louds')
+          if data not in robot.brain.get('louds_banned')
```

Comment:
Uh. I don't understand what deleteLoud means to you.

```coffeescript
            robot.brain.get('louds_banned').push(data)
```
I stored things into the `louds_banned` database, not deleting the entries from `louds`